Wiley Online Library.js

```diff
@@ -1183,59 +1225,6 @@ var testCases = [
 				"seeAlso": []
 			}
 		]
-	},
-	{
-		"type": "web",
-		"url": "http://onlinelibrary.wiley.com/doi/10.1002/14651858.CD009192.pub2/abstract",
```

Comment:
This URL now redirects to http://cochranelibrary-wiley.com/doi/10.1002/14651858.CD009192.pub2/abstract and we should invest what the status with the Cochrane Library is. But I suggest to open a new ticket after we have merged this.